src/github.ts

```diff
@@ -84,10 +84,25 @@ query lastCommitDate($owner: String!, $repo: String!, $ref: String!, $path: Stri
         )
     }
 
-    const {edges} = result.repository.ref.target.history
+    const {history} = result.repository.ref.target
```

Comment:
I test the code, `result.repository.ref` this is null, so `result.repository.ref.target` throws `cannot read property 'target' of null`

Reply:
Looks really strange. Seems Github's graphql API has unexpected behavior in some rare cases.

Reply:
Could you try again?

Reply:
Error: Unable to retrieve history for path 'libduckdb-sys' (res: {"repository":{"ref":null}})